packages/logtk/logtk.1.5.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+homepage: "https://github.com/sneeuwballen/zipperposition"
+synopsis: "Core types and algorithms for logic"
+build: [
+  ["dune" "build" "-p" name]
+  ["dune" "build" "@doc" "-p" name] {with-doc}
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "base-bytes"
+  "base-unix"
+  "zarith"
+  "containers" { >= "1.0" }
+  "iter" { >= "1.2" }
+  "menhir" {build}
+  "dune" { build }
+  "alcotest" {with-test}
+  "qcheck-core" {with-test & >= "0.9"}
+  "qcheck-alcotest" {with-test & >= "0.9"}
+  "ocaml" {>= "4.03"}
+]
+depopts: [
+  "msat" { >= "0.8" < "0.9" }
```

Comment:
```suggestion
  "msat" { >= "0.8" < "0.9" }
```
These constraints should be in the `conflicts` field. Constraints in the `depopts` field is most probably not what you want